data-raw/lifetables_sample.csv

```diff
@@ -0,0 +1,29 @@
+x,qx,lx,dx,Lx,Tx,ex,sex,year
+99,0.35368,335,119,276,703,2.1,M,1900
+79,0.04954,37833,1874,36895,358165,9.47,F,1900
+59,0.02185,61287,1339,60617,1087067,17.74,M,1910
+39,0.00283,78275,222,78164,3129636,39.98,F,1910
+19,0.00234,85960,201,85860,4504347,52.4,M,1920
+119,0.8427600000000001,0,0,0,0,0.6799999999999999,M,1920
```

Comment:
This number seems particularly troubling to me — where is the floating point inaccuracy coming from? Maybe we should `zapsmall()` these columns?

Reply:
This looks like a regular old floating point issue to me.  readr::parse_number and tidyr::extract_numeric aren't using identical code, so we end up with 

``` r
issue <- data.frame(a=readr::parse_number("0.68"),b=tidyr::extract_numeric("0.68")) 
#> extract_numeric() is deprecated: please use readr::parse_number() instead
readr::write_csv(issue, "/tmp/ouch")
original_digits <- getOption("digits")
options(digits = 20)
print(issue)
#>                        a                      b
#> 1 0.67999999999999993783 0.68000000000000004885
options(digits = original_digits)
```

<sup>Created on 2018-12-06 by the [reprex package](https://reprex.tidyverse.org) (v0.2.1)</sup>

If you want, I can zapsmall() these 🤷‍♂️ .  As far as I'm concerned this is fine as is, these CSVs are meant to help be diagnostic but aren't tests per se.

Reply:
Yeah, can you zapsmall please?

Reply:
Zapped.